### PR TITLE
Remove extra upvote weight + prevent stakeless upvotes in staked community

### PR DESCRIPTION
**Pull request by rbennettcw**
## Link to Issue
Closes: #8681
Closes: #8752

## Description of Changes
- Removes extra upvote weight from reactions
- Prevents user from casting stakeless votes in a staked community
- Updates UI to show toast message with error if insufficient stake

## Test Plan
- Create a community, don't buy stake
- Create a thread and attempt to upvote it– ensure that the upvote fails with toast
- Create a comment and attempt to upvote it– ensure that the upvote fails with toast
- Buy some stake
- Create a thread, ensure that thread upvote can be cast
- Create a comment, ensure that comment upvote can be cast

## Deployment Plan
N/A

## Other Considerations
N/A

### CODE REVIEW

File: packages/commonwealth/client/scripts/state/api/comments/createReaction.ts [hunk 1]
```diff
@@ -87,7 +88,14 @@ const useCreateCommentReactionMutation = ({
 
       return reaction;
     },
-    onError: (error) => checkForSessionKeyRevalidationErrors(error),
+    onError: (error) => {
+      if (error instanceof AxiosError) {
+        if (error.response?.data?.error?.toLowerCase().includes('stake')) {
```
Comment (Rotorsoft) [inline packages/commonwealth/client/scripts/state/api/comments/createReaction.ts:93]: is there a more type safe way to check for stake related errors?

File: packages/commonwealth/client/scripts/state/api/threads/createReaction.ts [hunk 2]
```diff
@@ -81,7 +82,14 @@ const useCreateThreadReactionMutation = ({
       userId &&
         markTrainingActionAsComplete(UserTrainingCardTypes.GiveUpvote, userId);
     },
-    onError: (error) => checkForSessionKeyRevalidationErrors(error),
+    onError: (error) => {
+      if (error instanceof AxiosError) {
+        if (error.response?.data?.error?.toLowerCase().includes('stake')) {
```
Comment (Rotorsoft) [inline packages/commonwealth/client/scripts/state/api/threads/createReaction.ts:87]: same as above

**Review by Rotorsoft — APPROVED** [review]
LGTM... just a comment about catching stake related errors that looks a little brittle

File: packages/commonwealth/client/scripts/state/api/comments/createReaction.ts (hunk 1 above)
Comment (rbennettcw) [reply packages/commonwealth/client/scripts/state/api/comments/createReaction.ts:93]: Yeah thought about that. Do we have a convention yet for sharing constants/errors between BE/FE?

Comment (Rotorsoft) [reply packages/commonwealth/client/scripts/state/api/comments/createReaction.ts:93]: we have libs/shared and errors can carry specific codes or other details that are better suited for this

**Review by masvelio — APPROVED** [review]

File: packages/commonwealth/client/scripts/state/api/comments/createReaction.ts (hunk 1 above)
Comment (rbennettcw) [reply packages/commonwealth/client/scripts/state/api/comments/createReaction.ts:93]: Will tackle this in a subsequent PR next time we need to share errors. Spent too much time on fixing the tests today.

**Pull request merged**